Readme.md

```diff
@@ -230,6 +230,7 @@ Calls the callback with a `Set` based on the options passed.  Options:
 
 - `set`: A set instance if you created your own
 - `stateStore`: A store instance to load and store migration state, or a string which is a path to the migration state file
+- `migrations`: An object where keys are migration file names and values are migration modules loaded with `require()`
```

Comment:
```suggestion
- `migrations`: An object where keys are migration names and values are migration objects
```

I think they can be loaded however you want, you can define them inline if you want. I don't think we need to specify "with require"

Reply:
Fair point.